src/test/java/net/sf/jabref/logic/xmp/XMPUtilTest.java

```diff
@@ -860,6 +861,28 @@ public void testWriteMultiple() throws IOException, TransformerException {
         assertEqualsBibtexEntry(t3BibtexEntry(), b);
     }
 
+    /**
+     * Tests whether a edit-protected PDF can be read
+     */
+    @Test
+    public void testReadProtectedPDFHasMetaData() throws Exception {
+        try (InputStream is = XMPUtilTest.class.getResourceAsStream("/pdfs/write-protected.pdf")) {
+            Assert.assertTrue(XMPUtil.hasMetadata(is));
+        }
+    }
+
+    /**
+     * Tests whether a edit-protected PDF can be read
+     */
+    @Test
+    public void testReadProtectedPDFHasCorrectMetaData() throws Exception {
+        try (InputStream is = XMPUtilTest.class.getResourceAsStream("/pdfs/write-protected.pdf")) {
+            List<BibEntry> bibentries = XMPUtil.readXMP(is);
+            Assert.assertEquals(1, bibentries.size());
+            BibtexEntryAssert.assertEquals(this.getClass(), "write-protected.bib", bibentries.get(0));
```

Comment:
Fixed :innocent:. - I've been in a mode that I read the resulting bib files and not interpret code. The code is ca. 6 lines longer, but we have one file less. Hope, that is OK now. https://github.com/JabRef/jabref/pull/942/commits/a39f11f8c4e66eef2921e16dbe01d325662228d4#diff-4538b7a42e8c15776dcbb34bca7c25d3R882